src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/QualityGuidelines/ReviewVisibleEventHandlersTests.cs

```diff
@@ -25,8 +25,12 @@ public class Program
     public void Handler1(object sender, EventArgs args) {}
     protected void Handler2(object sender, EventArgs args) {}
 }",
+#pragma warning disable RS0030 // Do not used banned APIs
                 VerifyCS.Diagnostic().WithLocation(6, 17).WithArguments("Handler1"),
+#pragma warning restore RS0030 // Do not used banned APIs
+#pragma warning disable RS0030 // Do not used banned APIs
```

Comment:
nit: redundant.

Reply:
I saw these, but figured they are fine since the cleanup pass will fix one file at a time.